src/js/player.js

```diff
@@ -2852,6 +2853,26 @@ class Player extends Component {
     this.trigger('exitFullWindow');
   }
 
+  /**
+   * Disable Picture-in-Picture mode.
+   *
+   * @param {boolean} value
+   *                  - true will disable Picture-in-Picture mode
+   *                  - false will enable Picture-in-Picture mode
+   */
+  disablePictureInPicture(value) {
+    if (value === undefined) {
+      return this.techGet_('disablePictureInPicture');
+    }
+    this.techCall_('setDisablePictureInPicture', value);
+    this.options_.disablePictureInPicture = value;
+    const PiPToggle = this.controlBar.pictureInPictureToggle;
+
+    if (PiPToggle) {
+      PiPToggle.handlePiPEnabledChange();
```

Comment:
what about triggering something like `disablepictureinpicturechanged` and having the PiPToggle listen to that in addition to loadedmetadata? That way we don't need to worry about getting the toggle directly and someone could use that for a custom pip toggle as well?

Reply:
Nice idea. I have just updated the PR.